src/nv_ingest/stages/nim/infographic_extraction.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import functools
+import logging
+import traceback
+from concurrent.futures import ThreadPoolExecutor
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import pandas as pd
+from morpheus.config import Config
+
+from nv_ingest.schemas.infographic_extractor_schema import InfographicExtractorSchema
+from nv_ingest.stages.multiprocessing_stage import MultiProcessingBaseStage
+from nv_ingest.util.image_processing.transforms import base64_to_numpy
+from nv_ingest.util.nim.helpers import NimClient
+from nv_ingest.util.nim.helpers import create_inference_client
+from nv_ingest.util.nim.paddle import PaddleOCRModelInterface
+
+logger = logging.getLogger(__name__)
+
+PADDLE_MIN_WIDTH = 32
+PADDLE_MIN_HEIGHT = 32
+
+
+def _update_metadata(
+    base64_images: List[str],
+    paddle_client: NimClient,
```

Comment:
Is this supposed to be paddle_client?

Reply:
Yes, we only need Paddle to extract test from the images. No need for another yolox model unlike table or chart extraction.